x-pack/legacy/plugins/snapshot_restore/public/app/components/policy_form/steps/step_logistics.tsx

```diff
@@ -347,7 +347,7 @@ export const PolicyStepLogistics: React.FunctionComponent<StepProps> = ({
           onChange={e => {
             updatePolicy(
               {
-                snapshotName: e.target.value.toLowerCase(),
+                snapshotName: e.target.value,
```

Comment:
@jkelastic I think we are on the right track with the proposed changes, I would just like to know what the reason was that we `.toLowerCase()`ed originally? Did we just never need to take this extra step?

Reply:
@jloleysens yes, we never need to take this step as `.toLowerCase()` on date format will negate MM to mm. The uppercase matters as MM is month and mm is minutes.

Reply:
@jkelastic see @alisonelizabeth 's comment below, it looks like we need to keep the names lowercase :)